news/2017-07-14-scorewatch-july-week-2.md

```diff
@@ -0,0 +1,48 @@
+
+<iframe width="100%" height="315" src="https://www.youtube.com/embed/7e4iJabYwpA" frameborder="0" allowfullscreen></iframe></br>
+
+He isn’t the only one to enter the 700pp club that day! [Yaong](https://osu.ppy.sh/u/1883865) surged into the 11k pp club with an unbelievable 98.82% HR FC on [xi - Blue Zenith](https://osu.ppy.sh/b/658127) to score 707pp and taking the No. 2 spot on the map, taking the first modded FC on the FOUR DIMENSIONS difficulty. You have to watch this.
+
+<iframe width="100%" height="315" src="https://www.youtube.com/embed/Sf29XhQc1cM" frameborder="0" allowfullscreen></iframe></br>
+
+[Rucker](https://osu.ppy.sh/u/147515) definitely had [MinG3012](https://osu.ppy.sh/u/1583218) (and [Axarious](https://osu.ppy.sh/u/2614511) and [Epiphany](https://osu.ppy.sh/u/3256299)) in his mind as he went on to take a somewhat unbelievable 16.90% NF FC on [IOSYS - Cirno no Perfect Sansuu Kyoushitsu](https://osu.ppy.sh/b/893739) to earn himself 118pp. So, when’s the all 50 run? We will be waiting.
+
+<iframe width="100%" height="315" src="https://www.youtube.com/embed/7iPh4um_Qwo" frameborder="0" allowfullscreen></iframe></br>
+
+osu!catch World Cup 2017 winner [[224]Hyperw7](https://osu.ppy.sh/u/4158549) definitely still has some adrenaline in his blood, unleashing the first HDHR FC on [onoken - P8107](https://osu.ppy.sh/b/686192r) with an amazing 99.88% accuracy to take 1158pp. So much patience was put into this, but it all paid off.
+
+<iframe width="100%" height="315" src="https://www.youtube.com/embed/DDrvck6gYmU" frameborder="0" allowfullscreen></iframe></br>
+
+There are no brakes to what [jakads](https://osu.ppy.sh/u/259972) can do, sniping cheetose by a mere 90 points for the No. 1 spot on [Kurokotei - Galaxy Collapse](https://osu.ppy.sh/b/992512) and earning 685pp with his 99.91% play. It may not be the highest accuracy on the leaderboards, but it certainly is a really great score.
+
+<iframe width="100%" height="315"  src="https://www.youtube.com/embed/1lUnxqYUf08" frameborder="0" allowfullscreen></iframe></br>
+
+NOTABLE MENTIONS
+-----
+
+Seeing an S on [Inferi - The Promethean Kings](https://osu.ppy.sh/b/1083470) is odd indeed, but not that odd for [-GN](https://osu.ppy.sh/u/895581), stealing the limelight by taking the first FC, using EZHT to guide his way to a 98.68% FC to earn 388pp. The patience in this is remarkable.
+
+A certain Monstrata map, [xi - Glorious Crown](https://osu.ppy.sh/s/558694), was recently ranked, and as such, [Angelsim](https://osu.ppy.sh/u/1777162) conquered the 2nd hardest difficulty in the mapset, scoring an impressive 98.54% HDHR FC to claim 552pp and the No. 1 spot. The final difficulty has yet to be conquered.
```

Comment:
wouldn't a listing to his ranked beatmaps be better? since we're talking of his maps, people aren't looking for his profile there i think